DESCRIPTION

```diff
@@ -52,9 +52,11 @@ Imports:
 Suggests:
     diffdf,
     DT,
+    htmltools,
```

Comment:
Is this package really needed?  I see it called one time??  or is it a dependency for reactable

Reply:
ACtually htmltools is a dependency of reactable.  Do we need to add it to the Suggests??
![image](https://github.com/pharmaverse/admiral/assets/10111024/762f6172-d463-4d5d-b023-28c39ada3307)

Reply:
R-CMD checks failed without `htmltools`. So I think we need it.

Reply:
I would keep `htmltools` and `reactable` in `Suggests:` for this release.

Next year we can investigate if we should move completely to `reactable`. I think for this release it's too late.

Reply:
okay...im very reluctant about this addition of packages.

I think we could of just used [kable](https://bookdown.org/yihui/rmarkdown-cookbook/kable.html) to display your tables as we already have knirtr.  

but ill add it to discussion